src/Resources/app/administration/src/module/findologic-module/components/findologic-config/index.js

```diff
@@ -146,29 +126,14 @@ Component.register('findologic-config', {
       }
     },
 
-    /**
-     * @public
-     */
-    getCategories () {
-      this.isLoading = true;
-
-      const translatedCategories = [];
-      this.categoryRepository.search(this.categoryCriteria, Shopware.Context.api).then((items) => {
-        this.term = null;
-        this.total = items.total;
-        items.forEach((category) => {
-          translatedCategories.push({
-            value: category.id,
-            name: category.name,
-            label: category.translated.breadcrumb.join(' > ')
-          });
-        });
+    onCategoryAdd(item) {
```

Comment:
Does the `item` has any API reference that we could add as a docblock? Would be easier for debugging when knowing how the structure looks like.

Reply:
This component event is not documented at all: https://component-library.shopware.com/components/sw-category-tree-field

Just saw it in the implementation of Shopware. It should be a category entity.